lib/miq/ref_docs.rb

```diff
@@ -61,39 +59,35 @@ def update
 
     def clone_or_update_repo
       if File.directory?(tmp_dir)
-        logger.info "Updating local ref docs repo"
-        shell "cd #{tmp_dir} && git pull origin"
+        update_repo
       else
-        logger.info "Cloning #{repo} to #{tmp_dir}"
-        shell "git clone #{repo} --depth 1 #{tmp_dir}"
+        clone_repo
       end
     end
 
     def setup_ascii_binder
       logger.info "Installing Ascii Binder"
       shell [
         "cd #{tmp_dir}",
-        "git checkout #{branch}",
+        "git checkout master",
         "#{bundler} install"
       ].join(" && ")
     end
 
     def build_ref_docs
       logger.info "Building ref docs"
-      shell [
-        "cd #{tmp_dir}",
-        "#{bundler} exec asciibinder build"
-      ].join(" && ")
+      shell "cd #{tmp_dir} && #{bundler} exec asciibinder package"
     end
 
     def sync_files
       if File.directory?("#{tmp_dir}/#{src_dir}") || debug?
         prep dst_dir
-        logger.info "Syncing files to #{dst_dir}"
-        cmd = "rsync -av "
-        cmd << exclude_files.map{|x| "--exclude '#{x}'"}.join(' ')
-        cmd << " #{tmp_dir}/#{src_dir}/* #{dst_dir}/"
-        shell cmd
+
+        branches.each do |branch|
+          rsync_copy(branch)
+        end
```

Comment:
asciibinder should create each of the docsets in separate directories by name, so I would expect you *could* just rsync the entire base directory instead of having to loop